### PR TITLE
introduced informaiton in html

### PR DESCRIPTION
**Pull request by lkthorn**
Kalendarium

### CODE REVIEW

**Comment by Zellis-K** [comment]
Title is missing in the document

File: html/kalendarium.html [hunk 1]
```diff
@@ -36,10 +36,29 @@
...
-            <li>Startsida / </li>
-            <li></li>
+            <li><a href="/index.html">Startsida</a> / </li>
+            <li>Kalendarium</li>
         </ul>
 
+        <h1>Aktuellt i kalendariet</h1>
+
+        <p>Här kan du se alla aktuella aktiviteter inom myndigheten.</p>
+
```
Comment (NovaBoman) [inline html/kalendarium.html:46]: Wrap these in `<section>`.

File: html/kalendarium.html [hunk 2]
```diff
@@ -36,10 +36,29 @@
...
+                    <p>Tid: start 16 november 2020 och sedan löpande fram till slutet av 2021. Plats: online, samt fysiskt i Sverige och i Ryssland. Hur: webbinarier, online-möten och fysiska träffar med erfarenhets- och kunskapsutbyte som bas.</p>
+            </section>
+
+            <section>
+                <h2><a href="https://www.konstnarsnamnden.se/default.aspx?id=25709">IASPIS Kitchen Talk: Silvia Franceschini</a></h2>
+                    <p>Datum: 2021-11-11</p>
+                    <p>Tid: 17.00 Plats: Konstnärsnämnden, IASPIS-köket Adress: Maria skolgata 83, Stockholm Språk: engelska</p>
+            </section>
+
+        </article>
```
Comment (NovaBoman) [inline html/kalendarium.html:60]: These are two separate `<article>`s.
```suggestion
<article>
                <h2><a href="https://www.konstnarsnamnden.se/default.aspx?id=24784">Cross polination: curatorial exchange across Sweden and Russia</a></h2>
                    <p>Datum: 2020-12-15</p>
                    <p>Tid: start 16 november 2020 och sedan löpande fram till slutet av 2021. Plats: online, samt fysiskt i Sverige och i Ryssland. Hur: webbinarier, online-möten och fysiska träffar med erfarenhets- och kunskapsutbyte som bas.</p>
</article>

<article>
                <h2><a href="https://www.konstnarsnamnden.se/default.aspx?id=25709">IASPIS Kitchen Talk: Silvia Franceschini</a></h2>
                    <p>Datum: 2021-11-11</p>
                    <p>Tid: 17.00 Plats: Konstnärsnämnden, IASPIS-köket Adress: Maria skolgata 83, Stockholm Språk: engelska</p>
</article>
```

**Review by NovaBoman — CHANGES_REQUESTED** [review]
- [x] Add `<meta name="description" content"">` in `<head>`.

- [x] Add `<title>`
- [x] Change path to logo `` in `<header>`

**Comment by Zellis-K** [comment]
Tror jag råkade assigna mig fel på den här requesten den är din va Licia?

File: html/kalendarium.html (hunk 1 above)
Comment (RicardoCastelbon) [reply html/kalendarium.html:46]: correct, it is what will be on the "hero"

File: html/kalendarium.html (hunk 2 above)
Comment (RicardoCastelbon) [reply html/kalendarium.html:60]: True they are 2 diffierent news/articles

**Pull request merged**